groups/string.yml

```
@@ -0,0 +1,2 @@
+name: String
```

Also rename this file to `strings` (for symmetry with "arrays")

```suggestion
name: Strings
```